### PR TITLE
Scene Loading & Editor Snapshot Unit Tests

### PR DESCRIPTION
**Pull request by SYBIOTE**
This pull request adds tests for the scene loading functionality in the codebase. It includes tests for loading an empty scene, loading a scene with a root entity, loading a scene with a child entity, and unloading a child entity.

### CODE REVIEW

File: packages/engine/src/scene/systems/SceneLoadingSystem.test.tsx [hunk 1]
```diff
@@ -0,0 +1,1380 @@
...
+import { FogSettingsComponent } from '../components/FogSettingsComponent'
+import { ModelComponent } from '../components/ModelComponent'
+import { NameComponent } from '../components/NameComponent'
+import { SceneAssetPendingTagComponent } from '../components/SceneAssetPendingTagComponent'
+import { UUIDComponent } from '../components/UUIDComponent'
+import { SceneLoadingSystem } from './SceneLoadingSystem'
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '1'
+
+const modelLink = '/packages/projects/default-project/assets/collisioncube.glb'
+const testScene = {
```
Comment (HexaField) [inline packages/engine/src/scene/systems/SceneLoadingSystem.test.tsx:54]: let's put this in a .json file and import it to improve readability of this file

File: packages/editor/src/functions/EditorControlFunctions.test.tsx [hunk 2]
```diff
@@ -0,0 +1,855 @@
...
+import { SceneDataType, SceneID, SceneJsonType } from '@etherealengine/engine/src/schemas/projects/scene.schema'
+import { UserID } from '@etherealengine/engine/src/schemas/user/user.schema'
+import { LocalTransformComponent } from '@etherealengine/engine/src/transform/components/TransformComponent'
+import { applyIncomingActions, getMutableState } from '@etherealengine/hyperflux'
+import { act, render } from '@testing-library/react'
+import React from 'react'
+import { FogSettingsComponent } from '../../../engine/src/scene/components/FogSettingsComponent'
+import { EditorControlFunctions } from './EditorControlFunctions'
+
+const testScene = {
```
Comment (HexaField) [inline packages/editor/src/functions/EditorControlFunctions.test.tsx:53]: let's put this in a .json file and import it to improve readability of this file

File: packages/editor/src/functions/EditorControlFunctions.ts [hunk 3]
```diff
@@ -140,6 +140,7 @@ const modifyProperty = <C extends Component<any, any>>(
   }
 
   for (const [sceneID, entities] of Object.entries(scenes)) {
+    console.log('sceneID', sceneID, entities)
```
Comment (HexaField) [inline packages/editor/src/functions/EditorControlFunctions.ts:143]: cleanup log

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/engine/src/scene/systems/SceneLoadingSystem.test.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/engine/src/scene/systems/SceneLoadingSystem.test.tsx:54]: roger that, ill put it right next to the file

**Review by HexaField — APPROVED** [review]

**Pull request merged**